language/testing-infra/e2e-tests/src/common_transactions.rs

```diff
@@ -21,7 +21,7 @@ pub static CREATE_ACCOUNT_SCRIPT: Lazy<Vec<u8>> = Lazy::new(|| {
     import 0x1.Diem;
     import 0x1.DiemAccount;
 
-    main<Token: store>(account: signer, fresh_address: address, auth_key_prefix: vector<u8>, initial_amount: u64) {
```

Comment:
@tzakian  @runtian-zhou 
Oh this shit is so haggard... and just getting kind of useless. We never run these "common transactions" in practice, so what's the point of testing them with each version. It's not a real scenario... so I don't think we need to test it...

We are handrolling a bunch of scripts, and then compiling them against the current releases. And then they blow up when they don't compile against all of the old releases! Like... no shit, lol

So his test fails at language/e2e-testsuite/src/tests/account_limits.rs:233:9, when it calls `execute_and_apply` with a writeset script for `encode_add_account_limits_admin_script`
This script is compiled against the current release, so of course it won't link with the V1 or V2 release!

We can't even just compile the script with the given version because the script needs to syntactically change depending on what release it is in! Madness!

I have no suggestions at the moment, but we can ponder it....

Reply:
Perhaps we can just kill these tests and move them into unit tests. these language-e2e-tests should be for testing real scenarios with real scripts/script functions. There doesn't make sense to be compiling new scripts to throw at all the versions.

Reply:
After sleeping on it, maybe for anything using custom scripts, we can kill using the test with anything but the current DF build. Thoughts @tzakian @runtian-zhou @vgao1996 ?

Reply:
I think switching this test over (and any others that use hand-rolled scripts) to only test against the current release of the DF makes sense to me. I'm happy to put up a quick PR making this change it that would be helpful.

Reply:
Make it so